weather-app/src/App.js

```diff
@@ -0,0 +1,142 @@
+import { useState } from "react";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
+
+function App() {
+  const [city, setCity] = useState('');
+  const [weatherInfo, setWeatherInfo] = useState(null);
+  const [forecastInfo, setForecastInfo] = useState('');
+
+  const apiKey = '8129711b603f61b7990132158538443b';
+
+  function getWeather() {
+    const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}&units=metric`;
```

Comment:
Ar fi fost frumos ca dacă sunt mai multe rezultate pentru căutare, să permiți userului să aleagă la ce se referă. 

De exemplu, San Jose este și un oraș mare american, dar și capitala Costa Ricii. Cel puțin se putea pune undeva în interfață ce oraș s-a găsit, pentru ca eventual utilizatorul să ajusteze căutarea.